packaging/makeself/install-or-update.sh

```diff
@@ -208,20 +219,50 @@ done
 
 # -----------------------------------------------------------------------------
 
-echo "Configure TLS certificate paths"
-if [ ! -L /opt/netdata/etc/ssl ] && [ -d /opt/netdata/etc/ssl ] ; then
-  echo "Preserving existing user configuration for TLS"
-else
+select_system_certs() {
   if [ -d /etc/pki/tls ] ; then
-    echo "Using /etc/pki/tls for TLS configuration and certificates"
+    echo "${1} /etc/pki/tls for TLS configuration and certificates"
     ln -sf /etc/pki/tls /opt/netdata/etc/ssl
   elif [ -d /etc/ssl ] ; then
-    echo "Using /etc/ssl for TLS configuration and certificates"
+    echo "${1} /etc/ssl for TLS configuration and certificates"
     ln -sf /etc/ssl /opt/netdata/etc/ssl
-  else
-    echo "Using bundled TLS configuration and certificates"
-    ln -sf /opt/netdata/share/ssl /opt/netdata/etc/ssl
   fi
+}
+
+select_internal_certs() {
+  echo "Using bundled TLS configuration and certificates"
+  ln -sf /opt/netdata/share/ssl /opt/netdata/etc/ssl
```

Comment:
This doesn't work because if a directory or symlink to a directory with the target name already exists, the symlink will be created inside it.

There is `-n` option

```
-n, --no-dereference
       treat LINK_NAME as a normal file if it is a symbolic link to a directory
```

but POSIX ln doesn't have it.

Reply:
```bash
# ln -sfv /opt/netdata/share/ssl /opt/netdata/etc/ssl
'/opt/netdata/etc/ssl/ssl' -> '/opt/netdata/share/ssl
```

Reply:
This should be fixed by the latest commit.